academy/models/models.py

```diff
@@ -0,0 +1,8 @@
+from odoo import fields, models
+
+
+class Teachers(models.Model):
+    _name = "academy.teachers"
```

Comment:
the name of the models is singular

```suggestion
    _name = "academy.teacher"
```

![image](https://github.com/AntonioMartinD/Building_a_website/assets/35231827/c24e4f0d-61d2-44a3-b8ea-17e84303e19a)